src/app/static/src/tests/components/plots/choropleth/choropleth.test.ts

```diff
@@ -575,10 +591,10 @@ describe("Choropleth component", () => {
 
         onEachFeatureFunction(mockFeature, mockLayer);
         expect(mockLayer.bindTooltip.mock.calls[0][0]).toEqual(`<div>
-                                <strong>Area 1</strong>
-                                <br/>1.00%
-                                <br/>(1.00% - 10.00%)
-                            </div>`);
+                        <strong>Area 1</strong>
+                        <br/>1.00%
+                        <br/>(1.00% - 10.00%)
+                    </div>`);
```

Comment:
I'm not really sure why the whitespace here has changed but figured it wouldn't make a difference to the rendered tooltip

Reply:
Oh yes that makes sense. Is that important? This will all be rendered nicely when we display the HTML right. Suppose maybe the test could strip whitespace and then check equality if the whitespace is uninteresting?

Reply:
oh i mean the html will ignore it, dont think it is important to worry about this whitespace, we dont really worry about it elsewhere and would just complicate testing a bit